js/tests/unit/util/index.spec.js

```diff
@@ -35,36 +35,6 @@ describe('Util', () => {
       expect(Util.getSelectorFromElement(testEl)).toEqual('.target')
     })
 
-    it('should get selector from href if no data-target set', () => {
```

Comment:
Shouldn't we move these tests to scrollspy?

Reply:
good catch 👍